ui/components/component-library/badge-wrapper/badge-wrapper.stories.js

```diff
@@ -0,0 +1,364 @@
+      style={{ alignSelf: 'flex-start' }}
+    >
+      <BadgeWrapper
+        badge={
+          <Tag
+            label="9999"
+            backgroundColor={Color.errorDefault}
+            labelProps={{ color: Color.errorInverse }}
+            borderColor={BorderColor.errorDefault}
+          />
+        }
+        anchorElementShape={BadgeWrapperAnchorElementShape.rectangular}
+      >
+        <Box
+          paddingTop={1}
+          paddingBottom={1}
+          paddingRight={8}
+          paddingLeft={8}
+          borderRadius={BorderRadius.SM}
+          borderColor={BorderColor.borderDefault}
+          backgroundColor={Color.backgroundDefault}
+        >
+          NFTs
+        </Box>
+      </BadgeWrapper>
+    </Box>
+  </Box>
+);
+
+export const Position = () => (
+  <Box display={DISPLAY.FLEX} gap={4}>
+    <BadgeWrapper
+      position={BadgeWrapperPosition.topLeft}
+      badge={
+        <AvatarNetwork
+          size={Size.XS}
+          name="Arbitrum One"
+          src="./images/avax-token.png"
+          borderColor={BorderColor.borderMuted}
+        />
+      }
+    >
+      <AvatarAccount
+        address="0x5CfE73b6021E818B776b421B1c4Db2474086a7e1"
+        size={Size.MD}
+      />
+    </BadgeWrapper>
+    <BadgeWrapper
+      badge={
+        <AvatarNetwork
+          size={Size.XS}
+          name="Arbitrum One"
+          src="./images/avax-token.png"
+          borderColor={BorderColor.borderMuted}
+        />
+      }
+    >
+      <AvatarAccount
+        address="0x5CfE73b6021E818B776b421B1c4Db2474086a7e1"
+        size={Size.MD}
+      />
+    </BadgeWrapper>
+    <BadgeWrapper
+      position={BadgeWrapperPosition.bottomLeft}
+      badge={
+        <AvatarNetwork
+          size={Size.XS}
+          name="Arbitrum One"
+          src="./images/avax-token.png"
+          borderColor={BorderColor.borderMuted}
+        />
+      }
+    >
+      <AvatarAccount
+        address="0x5CfE73b6021E818B776b421B1c4Db2474086a7e1"
+        size={Size.MD}
+      />
+    </BadgeWrapper>
+    <BadgeWrapper
+      position={BadgeWrapperPosition.bottomRight}
+      badge={
+        <AvatarNetwork
+          size={Size.XS}
+          name="Arbitrum One"
+          src="./images/avax-token.png"
+          borderColor={BorderColor.borderMuted}
+        />
+      }
+    >
+      <AvatarAccount
+        address="0x5CfE73b6021E818B776b421B1c4Db2474086a7e1"
+        size={Size.MD}
+      />
+    </BadgeWrapper>
+  </Box>
+);
+
+export const PositionObj = () => (
+  <Box display={DISPLAY.FLEX} alignItems={AlignItems.baseline} gap={4}>
+    <BadgeWrapper
+      badge={
+        <AvatarNetwork
+          size={Size.XS}
+          name="Arbitrum One"
+          src="./images/avax-token.png"
+          borderColor={BorderColor.borderMuted}
+        />
+      }
+      positionObj={{ top: 4, right: -8 }}
+    >
+      <AvatarAccount
+        address="0x5CfE73b6021E818B776b421B1c4Db2474086a7e1"
+        size={Size.SM}
```

Comment:
Can we make this default size to `size={Size.MD}` to keep it in sync with the other stories?

Reply:
Yes! Fixed here https://github.com/MetaMask/metamask-extension/pull/17851/commits/d01346d0cff09d59c6741d4c6b2e3e0bc7f36bd8